example/src/SampleApp/SampleApp.tsx

```diff
@@ -91,7 +91,10 @@ function SampleApp() {
         collections.push(testCollection);
     }
 
-    return <FirebaseCMSApp
+    return <div>
```

Comment:
Hi, we can't add any logic here. This is the code of a sample app, like the developer would use it. We can't ask them to wrap the main component and and add an extra empty div. If you need to add an empty div for the captcha, better do it programatically somewhere in the library code. 
Thanks!

Reply:
got it, will do. Should I render the captcha div from within FirebaseCMSApp?

Reply:
I think it would be better to do that in `FirebaseLoginView`